### ISSUE TITLE
AndroidX compatible?

### ISSUE THREAD

**Issue by jasonlaw** [larryaasen/upgrader#3]
Hi @larryaasen ,
Is this package ready for AndroidX? If not, do you have a plan to do so?

Thanks in advance.

**Comment by larryaasen** [comment]
@jasonlaw That is a good question, and I don't know much about AndroidX and packages at this point. I need to look into this more and determine what needs to be changed to support AndroidX.

**Comment by jasonlaw** [comment]
Hi @larryaasen ,

I am using this as reference: https://flutter.dev/docs/development/packages-and-plugins/androidx-compatibility

From your package reference, at least I see the package_info is not AndroidX compatible.
![image](https://user-images.githubusercontent.com/107820/57187584-0a127b80-6f24-11e9-93ac-b67871699d1a.png)

This is from the package_info released notes:
![image](https://user-images.githubusercontent.com/107820/57187594-23b3c300-6f24-11e9-9065-14969f7ec4a8.png)

Hope this help.

**Comment by larryaasen** [comment]
@jasonlaw Thanks for pointing that out. I am updating all of the packages in the pubspec.yaml file (actually 5 of them) to pickup the versions that now support AndroidX. I should have this ready soon.

**Comment by larryaasen** [comment]
I just published version 0.5.0 of this upgrader package at https://pub.dartlang.org/packages/upgrader. I updated dependent packages to AndroidX compatible versions.

@jasonlaw Please let me know if this runs in your AndroidX app.

**Comment by jasonlaw** [comment]
Hi @larryaasen ,

Thanks for the fast action! I will definitely try this in my app. Thanks again!

**Comment by stevenspiel** [comment]
Hey, great work. I'm excited to use this plugin.

There appears to be a compatibility issue with Flutter stable, which is currently using Dart SDK `2.1.2-dev.0.0.flutter-0a7dcf17eb`. I tried installing 0.5.0 today, but ran into this issue:

```
The current Dart SDK version is 2.1.2-dev.0.0.flutter-0a7dcf17eb.

Because upgrader >=0.5.0 depends on xml >=3.4.0 which requires SDK version >=2.2.0 <3.0.0, upgrader >=0.5.0 is forbidden.


So, because zboviq depends on upgrader ^0.5.0, version solving failed.
```

Which Flutter channel are you using?

**Comment by larryaasen** [comment]
@stevenspiel I published version 0.5.1 of this pack using Dart 2.2.0 and xml 3.4.0 which passed the tests on the Flutter dev branch. Hopefully, this will solve your problem.

**Comment by stevenspiel** [comment]
Great! Thank you!
